test/v2/client/test_client.go

```diff
@@ -49,11 +51,14 @@ type TestClient struct {
 	certVerifierAddressProvider *test.TestCertVerifierAddressProvider
 	disperserClient             clients.DisperserClient
 	payloadDisperser            *payloaddispersal.PayloadDisperser
-	relayClient                 clients.RelayClient
+	relayClient                 relayv2.RelayClient
 	relayPayloadRetriever       *payloadretrieval.RelayPayloadRetriever
 	indexedChainState           core.IndexedChainState
-	retrievalClient             clients.RetrievalClient
+	retrievalClient             validator.ValidatorClient
 	validatorPayloadRetriever   *payloadretrieval.ValidatorPayloadRetriever
+	// For fetching blobs from the validators without verifying or decoding them. Useful for load testing
+	// validator downloads with limited CPU resources.
+	onlyDownloadRetrievalClient validator.ValidatorClient
```

Comment:
nit: rename to `onlyDownloadValidatorClient`

Reply:
fixed